easybuild/easyconfigs/l/libxml2.Python/libxml2.Python-2.9.8-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'PythonPackage'
+
+name = 'libxml2.Python'
```

Comment:
@hajgato Do you mind changing this to `libxml2-python`?

Cfr. my naming used in #7261 (but your approach using `PythonPackage` and using `libxml2` as a dep is better/cleaner)

Reply:
@boegel Done!